tools/cut_a_release.md

```diff
@@ -45,22 +45,27 @@ between the crates, it must be done in specific order:
 - `deno_core` - all crates depend on `deno_core` so it must always be published
   first
 - `bench_util`
-- crates in `ext/` directory
-  - `deno_net`, `deno_websocket` and `deno_fetch` depend on `deno_tls`, so the
-    latter must be bumped and released first
-  - `deno_fetch`, `deno_crypto`, `deno_timers` and `deno_webstorage` depend on
-    `deno_web`, so the latter must be bumped and released first
-  - `deno_url` depends on `deno_webidl`, so the latter must be bumped and
-    released first
-  - `deno_timers` depends on `deno_url`, so the latter must be bumped and
-    released first
-  - `deno_http` depends on `deno_websocket`, so the latter must be bumped and
-    released first
+- crates in `ext/` directory, publish in the following order:
+  - broadcast_channel
+  - console
+  - ffi
+  - tls
+  - web
+  - webgpu
+  - webidl
+  - websocket
+  - webstorage
+  - crypto
+  - fetch
+  - http
+  - net
+  - url
+  - timers
```

Comment:
I think it's easier to just follow a list of the crates as it's a bit more systematic.